src/ptolemy/raster.py

```diff
@@ -304,8 +387,8 @@ def rasterize(
                 logger.info("Done with mask 3")
         elif strategy == "weighted":
             nodata = 0
-            dims += ["geometry"]
-            coords["geometry"] = [i for geom, i in geoms_idxs]
+            dims += [self.idxkey]
+            coords[self.idxkey] = [self.tags[i][1] for geom, i in geoms_idxs]
```

Comment:
I am not deep enough in the code to understand the significance of this change.

Reply:
it simply renames the key for shapes to the original name, so instead of having a coord named "geometry", it is now "iso_a3" or whatever was in the original shape file